pkg/docker/docker.go

```diff
@@ -88,6 +88,9 @@ func (d *DockerClient) BuildImage(buildFolder string) (string, error) {
 			if err != nil {
 				return "", errors.Wrap(err, "Error mapping JSON error message. Error in build.")
 			}
+			if strings.Contains(msg,"lstat") && strings.Contains(msg,"no such file or directory") {
+				msg += ", check that the specified folders in metadata/openshift.json match the file structure of the build";
+			}
```

Comment:
https://golang.org/pkg/os/#IsNotExist

Reply:
There may or may not be other cases of missing files that should not have this exact error message.